test/JDBC/expected/BABEL-4161-vu-verify.out

```diff
@@ -0,0 +1,24 @@
+create table #t4161(c1 int, c2 int)
```

Comment:
we should have add tests where temp table is not part of ENR and create index on them.